File: 프로그래머스/unrated/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.swift
import Foundation

func solution(_ t:String, _ p:String) -> Int {
    
     var answer = 0

    for i in 0 ... t.count - p.count {
        let subStr = t.dropFirst(i).prefix(p.count)
        if let comp = Int(subStr), let val = Int(p) {
            answer += comp - val <= 0 ? 1 : 0
        }
    }

    return answer
    
    
 
}

File: 프로그래머스/unrated/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    
    
    lt = len(t)
    lp = len(p)
    ip = int(p)
    
    ans = 0
    for i in range(lt-lp+1):
        
        if(ip >= int(t[i:i+lp])):
            
            ans +=1 
    
    
    
    

    return ans